setup.py

```diff
@@ -56,7 +56,7 @@ def _get_plugin_version() -> str:
     install_requires=[
         "dbt-spark>=1.8.0, <2.0",
         "dbt-core~=1.8.0",
```

Comment:
For consistency, could you also bump this pin?

```suggestion
        "dbt-core>=1.8.0, <2.0",
```